docs/en/observability/analyze-monitors.asciidoc

```diff
@@ -10,7 +10,7 @@ The monitor detail screen displays several panels of information.
 == Status panel
 
 The *Status* panel displays a summary of the latest information regarding your monitor.
-You can view its availability, monitor ID, type, and any assigned tags. You can click a
+View its availability, monitor ID, type, and any assigned tags. Click a
```

Comment:
I would keep the original as it's consistent of how we document what a user can do with the UI.